meson.build

```diff
@@ -2536,6 +2537,15 @@ if conf.get('HAVE_LIBCRYPTSETUP') == 1
                 install : true)
 endif
 
+executable(
```

Comment:
this should move above the `endif`

Reply:
> this should move above the `endif`

Integrity isn't linking against `libcrypt` so should we really not build it if it's not available?

Reply:
yes, it's the same for other generators